src/Prelude/Linear.hs

```diff
@@ -23,20 +23,98 @@
 
 module Prelude.Linear
   ( -- * Standard Types, Classes and Related Functions
-    ($)
-  , (&)
-  , (<*)
-  , foldl
-  , foldr
-  , const
-  , id
-  , seq
+    -- ** Basic data types
+    module Data.Bool.Linear
+  , Prelude.Char
+  , module Data.Maybe.Linear
+  , module Data.Either.Linear
+    -- * Tuples
+  , fst
+  , snd
   , curry
   , uncurry
+    -- ** Basic type classes
+  , module Data.Eq.Linear
+  , module Data.Ord.Linear
+  , Prelude.Enum (..)
+  , Prelude.Bounded (..)
+    -- ** Numbers
+  , Prelude.Int
+  , Prelude.Integer
+  , Prelude.Float
+  , Prelude.Double
+  , Prelude.Rational
+  , Prelude.Word
+  , module Data.Num.Linear
+  , Prelude.Real (..)
+  , Prelude.Integral (..)
+  , Prelude.Floating (..)
+  , Prelude.Fractional (..)
+  , Prelude.RealFrac (..)
+  , Prelude.RealFloat (..)
```

Comment:
These are all classes that we probably want to have a linear version of eventually. Let's keep track of this.